Simplenote/NoticeFactory.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+struct NoticeFactory {
+    static func linkCopied() -> Notice {
+        return Notice(message: Messages.copyLink, action: nil)
```

Comment:
I think this Notice should say `Link Copied` (right now it says `Copy Link`)

Reply:
Ah nice catch, there should be a "copy link" button title and a "Link copied" alert message, I was missing the second one.  Added